### DISCUSSION TITLE
Numbered Redstone Dust signal

### DISCUSSION THREAD

**Discussion by Hyrule-Rose** [BedrockTweaks/Files#268] [Ideas]
I don't really know how to go further detail with what I'm trying to suggest.
However instead of having the regular Texture for Redstone being placed. It would Instead show a signal number between 1 and 15 from the starting power source

**Comment by SquatchHunter** [comment]
I believe the old method for this relied on shaders which with Render Dragon we cannot do at present, someone had made a working PoC of it without shaders but was having issues having redstone lines still join and such AND also show the power level
with vanilla items being data driven soon, we hope blocks will be and we can revisit but for now I believe this is a "cant do" simply cause of the 15 texture limitation

**Reply by Hyrule-Rose** [reply]
Okay Quite understandable. Sorry for the trouble
